### ISSUE TITLE
ls coloring ideas

### ISSUE THREAD

**Issue by janek-warchol** [janek-warchol/selenized#1]
Though it's really a side-effect of unnatural terminal color bindings (i.e. light gray is put in light cyan color slot), I really like how the coloring in `ls` output looks like with solarized color palette: most filetypes are highlighted with different shades of gray instead of actual colors.  This makes them look more restrained than default settings.  I think it would be nice to use this consciously in selenized.

* hidden files could be gray (i.e. lighter than foreground color)
* directories could be bold and slightly darker than foreground
* ...

**Comment by janek-warchol** [comment]
Note: it may be a good idea to use https://github.com/seebi/dircolors-solarized

**Comment by dngray** [comment]
> Note: it may be a good idea to use https://github.com/seebi/dircolors-solarized

Yes that would be a good start, looking at my comparison in Termite and using your LS_COLORS suggestion https://github.com/jan-warchol/selenized/commit/d2287118d47a4a57bbe650de455c26a53306de2d

![LS_COLORS](https://user-images.githubusercontent.com/48640805/62819145-b3c76100-bb40-11e9-8421-e67c7727137c.png)

I observed this. The solarized, with solarized LS colors looks as it should. I think selenized with solarized LS_COLORS is almost there, really all that needs changing is the purple archives and the gray regular files.

The middle one, no LS_COLORS isn't all that great. Particularly as there's no differentiation between broken symlinks, (something I find really useful). The coloring of audio/video files is nice too.

**Comment by dngray** [comment]
I came across this project which might be handy:

> [*vivid*](https://github.com/sharkdp/vivid) is a generator for the **`LS_COLORS`** environment variable that controls the colorized output of [`ls`](https://www.gnu.org/software/coreutils/manual/html_node/ls-invocation.html#ls-invocation), [`tree`](http://mama.indstate.edu/users/ice/tree/), [`fd`](https://github.com/sharkdp/fd), [`bfs`](https://github.com/tavianator/bfs), [`dust`](https://github.com/bootandy/dust) and many other tools.
